tests/kafkatest/tests/core/round_trip_fault_test.py

```diff
@@ -99,7 +99,7 @@ def test_produce_consume_with_latency(self):
         time.sleep(2)
         node_specs = {}
         for node in self.kafka.nodes + self.zk.nodes:
-            node_specs[node.name] = {"latencyMs": 500, "networkDevice": "eth0"}
+            node_specs[node.name] = {"rateLimitKbit": "3000", "latencyMs": 100, "networkDevice": "eth0"}
```

Comment:
Is there a way to parameterize these? It would be nice to pass the `latency` and `rate limit` when invoking system test. Allows us to simulate different scenario w/o updating code.

Reply:
done